src/pages/Details.js

```diff
@@ -2,83 +2,67 @@ import React from 'react';
 import '../styling/Details.css'
 import Source from '../components/Source'
 import { dateToString } from '../util';
-import Carousel from 'react-bootstrap/Carousel' 
+import Carousel from 'react-bootstrap/Carousel'
 import logo from '../images/cwb-logo-reverse-w-tagline 1.png';
 import axios from 'axios';
 import { withRouter } from 'react-router-dom';
-
-
+ 
+ 
 class Details extends React.Component {
```

Comment:
same comment as above -- change to functional component. `componentDidMount` can be replaced by `useEffect` with an empty dependencies array `[]` (reference if needed: https://reactjs.org/docs/hooks-effect.html)